identity/resource_user_test.go

```diff
@@ -20,6 +20,20 @@ func TestResourceUserRead(t *testing.T) {
 					ID:          "abc",
 					DisplayName: "Example user",
 					UserName:    "me@example.com",
+					Entitlements: []entitlementsListItem{
+						{
+							Value: AllowWorkspaceAccessEntitlement,
+						},
+						{
+							Value: AllowSQLAnalyticsAccessEntitlement,
+						},
+						{
+							Value: AllowInstancePoolCreateEntitlement,
+						},
+						{
+							Value: AllowClusterCreateEntitlement,
+						},
+					},
```

Comment:
why this much was added?.. refactoring should have added only an item with `AllowWorkspaceAccessEntitlement`.

Reply:
was missing coverage in lines so i added it i can remove it if needed.

Reply:
ah, okay. it's more that refactoring the code should never change the tests.